.circleci/config.yml

```diff
@@ -55,20 +55,90 @@ jobs:
             pip install --upgrade pip setuptools
             pip install -r requirements.txt
             pip install -r deid/requirements.txt
+            pip install -r dev_requirements.txt
       - run:
           name: Show bashrc file
           command: cat ${BASH_ENV}
       # Test
       #   This would typically be a build job when using workflows, possibly combined with build
       - run:
           name: Run unit tests
+          working_directory: ~/curation
+          command: |
+            mkdir -p tests/results/coverage/unit/xml
+            mkdir -p tests/results/coverage/unit/html
+            mkdir -p tests/results/junit/unit
+            ./tests/run_tests.sh -s unit
+          no_output_timeout: 300s
+          when:  always
+      - run:
+          name: Run integration tests
+          working_directory: ~/curation
           command: |
             t=$(git log -1 --pretty=%B)
-            if [[ "$t" == *"fast tests"* ]] ; then export ALL_TESTS="False" ; else export ALL_TESTS="True" ; fi
-            ./test/run_tests.sh
+            if [[ "$t" == *"all tests"* ]] || [[ ! -z "${CIRCLE_PULL_REQUEST}" ]] || [[ ! -z "${CIRCLE_PULL_REQUESTS}" ]];
```

Comment:
nit: `-n` is probably a more standard way to write this over `! -z`

Reply:
Done!  I don't spend a lot of time in bash.  Thanks.